src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int supply;
+    private int buy;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readFromFIle(fromFileName);
+        writeToFIle(toFileName);
+    }
+
+    private void readFromFIle(String fromFile) {
+        try (BufferedReader readFile = new BufferedReader(new FileReader(fromFile + ""))) {
```

Comment:
```suggestion
        try (BufferedReader readFile = new BufferedReader(new FileReader(fromFile))) {
```
Why did you need that?